src/nnfusion/core/graph/graph.cpp

```diff
@@ -409,6 +413,93 @@ void Graph::set_temporary_pool_size(size_t size)
     m_temporary_pool_size = size;
 }
 
+bool Graph::serialize_to_file(const std::string& file_path)
+{
+    nnfusion::serialize::GraphDef graphdef;
+    auto nnfusion_nodes = get_ordered_ops(true);
+    for (auto& nnfusion_node : nnfusion_nodes)
```

Comment:
Is it general enough to support operators with different attributes (e.g., Dot without transposes and Dot with transposes)? If so, could it be reused to generate unique identifiers for operators?

Reply:
@xysmlx hi,
as `L422` shown, we don't support the nnfusion nodes with attributes because of engineering efforts while I think `src/nnfusion/common/serialize/attr_value.proto` could hold different attributes.